lib/ui/lerp.dart

```diff
@@ -6,15 +6,23 @@
 
 part of dart.ui;
 
-/// Linearly interpolate between two numbers.
-// TODO(cbracken): Consider making a and b non-nullable.
-//                 https://github.com/flutter/flutter/issues/64617
+/// Linearly interpolate between two numbers, `a` and `b`, by an extrapolation
+/// factor `t`.
+///
+/// When both `a` and `b` are null, infinite, or NaN, `a` is returned.
```

Comment:
This line is confusing. I thought for a minute that it meant that `lerpDouble(null, double.infinity, 1.0)` would return null.

Reply:
Hrm -- indeed. Re-worded a little. Let me know if that still parses badly.